DeveloperNotes.md

```diff
@@ -1,5 +1,7 @@
 #ToDo
 
+* make open site open site (for env)
```

Comment:
Not 100% sure what this means.

Reply:
There's a visit site button..
![image](https://user-images.githubusercontent.com/12608907/108054586-c2041080-7046-11eb-85b7-223b7bb972cf.png)

We should make it open the current environment's National Careers site.